localization/french.cfg

```diff
@@ -0,0 +1,48 @@
+{
+	"reservedArray": ["si", "alors", "sinon", "fin", "répéter", "tant que", "lire", "écrire"],
```

Comment:
Unfortunately the parser does not support blanks in reserved words. Is it possible to change tant que to be a single word? If not could you please add an underscore making it tant_que.

Google translate suggest jusqu'à as a translation for until. Not sure if it is appropriate.